task_manager/models/user.py

```diff
@@ -1,16 +1,38 @@
-from task_manager import db
 from sqlalchemy import Column, Integer, String
+from werkzeug.security import generate_password_hash, check_password_hash
+
```

Comment:
since we're using flask-login, we probably shouldnt use werkzeug since we can use the flask bcrypt built in

Reply:
i'll see if i can refactor

Reply:
I searched this, and I don't think flask-login includes bcrypt. Maybe not worth the hassle since this also breaks my test database.

Reply:
bcrypt is a lot more secure, we can save this for later. You would probably just have to create new data using bcrypt